web/ui/module/codemirror-promql/src/client/prometheus.ts

```diff
@@ -117,6 +122,7 @@ export class HTTPPrometheusClient implements PrometheusClient {
       return this.fetchAPI<string[]>(request.uri, {
         method: this.httpMethod,
         body: request.body,
+        headers: this.requestHeaders,
```

Comment:
instead of adding each time the headers, why not adding it directly in `fetchAPI` ?

Reply:
Thanks, that seems better indeed. Adjusted & rebased!